processor/src/test/java/org/mapstruct/ap/test/bugs/_1482/SourceTargetMapper.java

```diff
@@ -0,0 +1,40 @@
+/**
+ *  Copyright 2012-2017 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.test.bugs._1482;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public abstract class SourceTargetMapper {
+
+    public static final SourceTargetMapper INSTANCE = Mappers.getMapper( SourceTargetMapper.class );
+
+    @Mapping( target = "bigDecimal", source = "wrapper" )
+    abstract Target map(Source source);
```

Comment:
What about the reverse case? From what I could see in the coverage report we are only testing when `assignability` is `Assignability.VISITED_ASSIGNABLE_TO`?

Reply:
Good point. Can't fix it right now but will have a look tomorrow early in the evening

Reply:
If it's blocking the release, you can merge it and I'll add a until test later tomorrow

Reply:
If it's blocking the release, you can merge it and I'll add a until test later tomorrow

Reply:
No problem. dunno if I manage this evening. There's a drinking event 🍻 . They're risky. 😄